packages/react/src/components/LinkList/LinkList.js

```diff
@@ -58,7 +58,7 @@ LinkList.propTypes = {
   /**
    * Describes heading of LinkList.
    */
-  heading: PropTypes.string.isRequired,
+  heading: PropTypes.string,
```

Comment:
According to the [func specs](https://ibm.ent.box.com/file/664463554207), `heading` is required.

```suggestion
  heading: PropTypes.string.isRequired,
```

Reply:
Good point - Do you think we should remove the [existence check for `heading`](https://github.com/carbon-design-system/ibm-dotcom-library/blob/v1.8.0/packages/react/src/components/LinkList/LinkList.js#L31)?

Reply:
Yes please, good catch @asudoh thanks!

Reply:
Cool, thanks @kennylam for clarifying! - Updated.